lib/happo/public/HappoDiffs.jsx

```diff
@@ -81,20 +81,66 @@ function HappoNewImages({ images }) {
   );
 }
 
-function HappoDiff({ image }) {
-  return (
-    <div>
-      <HappoImageHeading
-        image={image}
-      />
-      <div className='happo-diff__images'>
-        <img src={image.previous} />
+const HappoDiff = React.createClass({
+  propTypes: {
+    image: PropTypes.shape(imageObjectStructure),
+  },
+
+  getInitialState() {
```

Comment:
It would be nice to split this component into a container component that handles the state and a presentational component that renders the DOM.